app/Http/Requests/StoreUser.php

```diff
@@ -25,11 +26,12 @@ public function rules()
     {
         if (in_array($this->method(), ['POST'])) {
             return [
-                'firstname' => 'required|string|max:255',
-                'lastname'  => 'required|string|max:255',
-                'email'     => 'required|email|max:255|unique:users,email',
-                'username'  => 'required|string|max:255|unique:users,username',
-                'password'  => 'required|string|max:255'
+                'firstname'             => 'required|string|max:255',
+                'lastname'              => 'required|string|max:255',
+                'email'                 => 'required|email|max:255|unique:users,email',
+                'username'              => 'required|string|max:255|unique:users,username',
+                'password'              => ['required','string', 'confirmed','regex:/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*(_|[^\w])).+$/','min:8','max:20'],
+                'password_confirmation' => 'required|string|max:255'
```

Comment:
you don't need this validation

Reply:
do you mean password_confirmation one? or the regex?

Reply:
https://github.com/THM-Health/PILOS/pull/23/commits/1a33a7aa89dddde61afd7c0994421ca8d1e20bfb removed ``password_confirmation``